internal/npm_install/test/golden/@angular/core/BUILD.bazel.golden

```diff
@@ -677,7 +677,7 @@ node_module_library(
 node_module_library(
     name = "core__contents",
     srcs = [":core__files"],
-    scripts = [
+    named_sources = [
```

Comment:
sources seems like a strange name here, should be named_modules ? named_module_srcs ?

Reply:
I like `named_module_srcs` out of those two. Its fairly clear from that name that these are a subset of `srcs`.